lib/googleauth/base_client.rb

```diff
@@ -37,6 +37,8 @@ def apply! a_hash, opts = {}
           hash = Digest::SHA256.hexdigest token
           Google::Logging::Message.from message: "Sending auth token. (sha256:#{hash})"
         end
+
+        a_hash
```

Comment:
@lucasvianna Actually, I should have looked at this more carefully. Prior to [the change you referenced](https://github.com/googleapis/google-auth-library-ruby/pull/490/files#diff-c93d457c6c019fd581a2e9868339f63d1d24479d65971b5c3e4dfe559c59debbR34-R39) in #505, this method returned not the hash but the header value (i.e. the string `"Bearer #{token}"`). If you wanted to restore the original behavior, we'd need to do something different here. So just want to confirm whether you want the string or the full hash.

Also, on thinking about this, `apply!` isn't meant for you to use the return value. That's what `apply` is for. I'm happy to make a change here to restore the original behavior because it doesn't hurt anything, but I also recommend not depending on behavior that isn't actually documented in the method comments.

Reply:
@dazuma you're right. I changed it to reflect the old behavior. That's what I meant since the beginning. Thank you for pointing that out.

Also, I'll do what you recommend and use `apply` instead.